Sources/Build/llbuild.swift

```diff
@@ -92,7 +100,7 @@ public struct LLBuildManifestGenerator {
     }
 
     /// Create link command for products.
-    private func createLinkCommand(_ buildProduct: ProductBuildDescription) -> Command {
+    private func createLinkTarget(_ buildProduct: ProductBuildDescription) -> Target {
```

Comment:
Not sure I understand you. Here are the list of llbuild targets:

* One `main` target to build everything.
* One `test` target to build and test everything.
* One llbuild target for each SwiftPM target that depends on the `.swiftmodule` file.
* One llbuild target for each SwiftPM product that depends on the executable or lib file.